### DISCUSSION TITLE
Improve readme.md

### DISCUSSION THREAD

**Discussion by ariskemper** [open-circle/valibot#305] [Ideas]
- Idea is to improve Readme.md to be more informative
- Table of contents
- Having multiple sponsorships (gold, silver, bronze) so that companies could get their logo and link to their website which improves also their SEO
- Add coverage badge

**Comment by fabian-hiller** [comment]
Thanks for creating this discussion. I like the coverage badge idea. I will give you feedback on the other points next week.

**Comment by fabian-hiller** [comment]
As I said, I like the coverage badge idea. I am not sure about adding a table of contents. What points would you add there?

In order to secure funding in the long term, I would probably set up Open Collective next year and add the logos of sponsors to the website and readme. However, this is not a high priority at the moment, but it will become more important once we release v1.

**Reply by ariskemper** [reply]
Idea is to optimize readme.md for SEO, so table of contents would help in this matter  to include relevant keywords like name of validation functions etc, when you land first time on the repo you don't know what kind of validations valibot supports?

**Reply by fabian-hiller** [reply]
I understand that, but I am not sure if it is necessary to optimize the repository for SEO. Our website already ranks pretty well there without any tweaks.

**Reply by ariskemper** [reply]
@fabian-hiller more traffic from different sources repository + website is better, because it converts to more users/companies using the library. Besides that readme.md content is also listed on npm repository.

**Reply by fabian-hiller** [reply]
That's true, but I'm not sure an SEO optimized readme changes that much. I think the most important thing for a wider adoption is that people talk about it on social media and create content about Valibot that links to our repository and website. I will focus more on that at the time we reach v1.
